ipaclient/install/timeconf.py

```diff
@@ -82,12 +82,10 @@ def sync_chrony():
     # Restart chronyd
     services.knownservices.chronyd.restart()
 
-    sync_attempt_count = 3
+    sync_attempt_count = 2
     # chrony attempt count to sync with configiured servers
-    # each next attempt is tried after 10seconds of timeot
-    # 3 attempts means: if first immidiate attempt fails
-    # there is 10s delay between next attempts
-
+    # each next attempt is tried after 10 seconds of timeout
+    # 2 attempts means: if first attempt fails there is a 10s delay
     args = [paths.CHRONYC, 'waitsync', str(sync_attempt_count), '-d']
```

Comment:
The change makes sense. However, how about keeping 3-4 attempts but with a much lower interval?
From what I understand this would allow choosing another host to sync from but I only have had a cursory look at chronyc sources. 
The full syntax for selecting an interval is:
chronyc waitsync [max-tries [max-correction [max-skew [interval]]]]

The lowest interval chronyc accepts is 0.1.

Reply:
Sounds like a good idea. ``chronyc waitsync 6 0 0 2`` would perform 6 reqests (5 retries) with a 2 second interval and default correction / skew values. Total runtime is little more than 10 seconds.

Reply:
I have changed the values to 4 attempts with 3 second interval.